samples/js/node/package.json

```diff
@@ -9,7 +9,7 @@
     "https-proxy-agent": "^7.0.2",
     "opencv-wasm": "^4.3.0-10",
     "@tensorflow/tfjs-node": "^4.19.0",
-    "openvino-node": "^2024.3.0"
+    "openvino-node": "^2024.4.0"
```

Comment:
version on master should be aligned with 
https://github.com/akladiev/openvino/blob/master/src/core/include/openvino/core/version.hpp#L22-L24

Reply:
Disagree. If do this, it will fail, because `openvino-node@2024.5.0` package doesn't exist.
This commit it's update version commit in release branch. It must present in the master.